2024-08-21 16:23:09,849 - INFO - We will use 90% of the memory on device 0 for storing the model, and 10% for the buffer to avoid OOM. You can set `max_memory` in to a higher value to use more memory (at your own risk).
2024-08-21 16:23:16,718 - INFO - Initialized LLMZip with model: llama_3 and compression method: AC
2024-08-21 16:23:16,718 - INFO - Starting zipping process for file: Data/small_txt copy.txt
2024-08-21 16:23:16,718 - INFO - Using device: NVIDIA A10G
2024-08-21 16:23:16,719 - INFO - Starting compression for file: Data/small_txt copy.txt using model: llama_3
2024-08-21 16:23:16,722 - INFO - Tokenization truncated to 500 tokens.
2024-08-21 16:23:16,726 - INFO - Number of characters: 483, Number of tokens: 483, Ratio: 1.0
2024-08-21 16:23:16,726 - INFO - Initial input shape: torch.Size([1, 1])
2024-08-21 16:46:28,359 - INFO - Compression completed. Compressed data size: 40 bytes.
2024-08-21 16:46:28,360 - ERROR - An error occurred: dict contains fields not in fieldnames: 'Time (mins)'
2024-08-21 16:46:28,361 - ERROR - Traceback (most recent call last):
  File "/teamspace/studios/this_studio/LLM_Refactored/main.py", line 34, in main
    llmzip.zip(input_file, output_file)
  File "/teamspace/studios/this_studio/LLM_Refactored/llmzip.py", line 107, in zip
    AC_compress_file(self.model, self.model_name, self.tokenizer, input_path, output_path, 500)
  File "/teamspace/studios/this_studio/LLM_Refactored/Arithmetic_Coder.py", line 104, in AC_compress_file
    save_to_csv(model_name, language, num_characters, num_tokens, ratio, entropy, compression_ratio, redundancy, time_taken)
  File "/teamspace/studios/this_studio/LLM_Refactored/Arithmetic_Coder.py", line 24, in save_to_csv
    writer.writerow({
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/csv.py", line 154, in writerow
    return self.writer.writerow(self._dict_to_list(rowdict))
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.10/csv.py", line 149, in _dict_to_list
    raise ValueError("dict contains fields not in fieldnames: "
ValueError: dict contains fields not in fieldnames: 'Time (mins)'

